.github/file-paths.yaml

```diff
@@ -25,7 +25,7 @@ frontend_sources: &frontend_sources
 
 frontend_specs: &frontend_specs
   - *shared_specs
-  - "frontend/test/!(__support__|__runner__)/**"
+  - "frontend/test/**"
```

Comment:
This change alone makes https://github.com/metabase/metabase/pull/28366 obsolete